### PR TITLE
new reading and code along

### LINKED ISSUE #1: in progress
For content that will be included, refer to outline in Readme

deadline: 9/15/2015

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
closes #1 

need some :eyes: 

@dfenjves @ipc103 @deniznida @aviflombaum @AnnJohn

### CODE REVIEW

**Comment by AnnJohn** [comment]
@dfenjves And this one too to review?

**Comment by dfenjves** [comment]
Yup. may need an extra day to get to it

**Comment by AnnJohn** [comment]
Assigned to @dfenjves for review

Deadline: 9/11/2015

File: README.md [hunk 1]
```diff
@@ -1,16 +1,235 @@
...
+These are some of the kinds of questions SQL allows us to ask our database about this table, which we will cover later. 
+
+### Note on Column Names
+
+When we name columns in our database, there are a couple conventions we will follow. The first is that we will always use lowercase letters when refering to columns in our database. SQLite isn't case sensitive about its commands or column names, as we will discuss below, but it is general best practice for us to stick to lowercase for our column names.
+
+The Second convention we want to follow is more important. That is that when we have multiple words in a column name we link the together use underscores rather than spaces. We call this convention "snake_case". So, for instance, if we wanted to be more specific with our email column above, we would have called it something like email_address. If we wanted to split up name to first and last we might have columns called first_name and last_name.
+
+## Database Tables
+
```
Comment (dfenjves) [inline README.md:36]: Is there any setup on the computer that needs to be done (bundle install, etc)?

**Comment by AnnJohn** [comment]
@SophieDeBenedetto ^^

File: README.md (hunk 1 above)
Comment (SophieDeBenedetto) [inline README.md:36]: no set up required. this is a code along and they are instructed to work in the command line. the prev. reading had them install sqlite3 if they didn't have it already.

**Comment by dfenjves** [comment]
Looks great. Merging.

**Pull request merged**